docs/src/glossary.md.in

```diff
@@ -678,14 +684,14 @@ page](record-heterogeneity.md#sparse-data).
 
 A [keyword](#keyword) in the
 [Miller programming language](miller-programming-language.md)
-for [print, dump, and tee statements](http://127.0.0.1:8000/reference-dsl-output-statements.md#tee-statements)
-indicating that data are to be sent to the [_standard output_](https://en.wikipedia.org/wiki/Standard_streams#Standard_output_(stderr)).
+for [print, dump, and tee statements](reference-dsl-output-statements.md#tee-statements)
+indicating that data are to be sent to the [_standard error_](https://en.wikipedia.org/wiki/Standard_streams#Standard_error_(stderr)).
 
 ## stdout
 
 A [keyword](#keyword) in the
 [Miller programming language](miller-programming-language.md)
-for [print, dump, and tee statements](http://127.0.0.1:8000/reference-dsl-output-statements.md#tee-statements)
+for [print, dump, and tee statements](reference-dsl-output-statements.md#tee-statements)
```

Comment:
wow it's embarrassing how many of these i missed :(

Reply:
There are too many things to check. I'm not surprised. This one in particular is hidden until you try to follow the link, so is even easier to miss.